clinic/clinic-introduction.md

```diff
@@ -7,34 +7,35 @@ summary: Learn about the PingCAP Clinic Diagnostic Service (PingCAP Clinic), inc
 
 PingCAP Clinic Diagnostic Service (PingCAP Clinic) is a diagnostic service provided by PingCAP for TiDB clusters that are deployed using either TiUP or TiDB Operator. This service helps to troubleshoot cluster problems remotely and provides a quick check of cluster status locally. With PingCAP Clinic, you can ensure the stable operation of your TiDB cluster for its full life-cycle, predict potential problems, reduce the probability of problems, troubleshoot cluster problems quickly, and fix cluster problems.
 
-PingCAP Clinic is currently in the Beta testing stage for invited users only. This service provides the following two components to diagnose cluster problems:
+PingCAP Clinic is currently in the Technical Preview stage. This service provides the following two components to diagnose cluster problems:
 
-- Diag: a diagnostic tool deployed on the cluster side. Diag is used to collect cluster diagnostic data, upload diagnostic data to the Clinic Server, and perform a quick health check locally on your cluster. For a full list of diagnostic data that can be collected by Diag, see [PingCAP Clinic Diagnostic Data](/clinic/clinic-data-instruction-for-tiup.md).
+- Diag client:
+
+    Diag client (Diag) is a diagnostic tool deployed on the cluster side. Diag is used to collect cluster diagnostic data, upload diagnostic data to the Clinic Server, and perform a quick health check locally on your cluster. For a full list of diagnostic data that can be collected by Diag, see [PingCAP Clinic Diagnostic Data](/clinic/clinic-data-instruction-for-tiup.md).
 
     > **Note:**
     >
-    > - Diag temporarily **does not support** collecting data from the clusters deployed using TiDB Ansible.
-    > - For the PingCAP Clinic Beta version, if you want to upload data to the Clinic Server for remote troubleshooting using Diag, you need to contact [PingCAP technical support](https://en.pingcap.com/contact-us/) to get a trial account first.
+    > Diag temporarily **does not support** collecting data from the clusters deployed using TiDB Ansible.
 
-- Clinic Server: a cloud service deployed in the cloud. By providing diagnostic services in the SaaS model, the Clinic Server can not only receive uploaded diagnostic data but also work as an online diagnostic environment to store data, view data, and provide cluster diagnostic reports.
+- Clinic Server:
 
```

Comment:
This part is only for global users, which is the same part as line 12 in [clinic/clinic-data-instruction-for-tiup.md](https://github.com/pingcap/docs/pull/8035/files#diff-ac8b043e1a5c420338d8612be46e50dc1330368b461a4dd38ac8b44a870e6616).